api/models/I18nTranslation.js

```diff
@@ -81,6 +87,12 @@ module.exports = {
   },
 
   beforeUpdate: function (valuesToUpdate, cb) {
+    // Manual validation for 'value'
+    if (Object.prototype.hasOwnProperty.call(valuesToUpdate, 'value') && valuesToUpdate.value === null) {
```

Comment:
I think [ `Object.hasOwn`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) is a modern replacement for `Object.prototype.hasOwnProperty`.